tests/integration_tests/test_hyperopt.py

```diff
@@ -379,17 +386,33 @@ def test_hyperopt_run_hyperopt(csv_filename, search_space, tmpdir, ray_cluster):
     assert isinstance(hyperopt_results, HyperoptResults)
 
     # check for existence of the hyperopt statistics file
-    assert fs_utils.path_exists(os.path.join(tmpdir, experiment_name, HYPEROPT_STATISTICS_FILE_NAME))
+    with use_credentials(minio_test_creds()):
+        assert fs_utils.path_exists(os.path.join(tmpdir, experiment_name, HYPEROPT_STATISTICS_FILE_NAME))
+        for trial in hyperopt_results.experiment_analysis.trials:
+            assert fs_utils.path_exists(os.path.join(tmpdir, experiment_name, f"trial_{trial.trial_id}"))
```

Comment:
Is it also possible to add additional checks for ensuring that `model_hyperparameters.json` and `training_set_metadata.json` are also in this folder / exist / are non-empty? Just to make sure this works for hyperopt end-to-end since those files are super important.

Reply:
Yes, we can check those as well. Let's do that in a follow-up.